web/third/bootstrap-typeahead/typeahead.js

```diff
@@ -126,7 +126,7 @@ import {get_string_diff} from "../../src/util";
     this.sorter = this.options.sorter || this.sorter
     this.highlighter = this.options.highlighter || this.highlighter
     this.updater = this.options.updater || this.updater
-    this.$container = $(this.options.container).appendTo('body')
+    this.$container = this.options.$container || $(this.options.container).appendTo('body')
```

Comment:
I think rather than making the default `$container` override-able, which is easy to confuse with `container`, we can instead have an optional parameter, say `parentElement`, to which we append it. 

So, this line should then read something like: `this.$container = $(this.options.container).appendTo(this.options.parentElement || 'body')`

Reply:
I like that!

Reply:
done :)